### DISCUSSION TITLE
Allow users to customize the role heirarchy

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Capabilities#671] [Feature Requests]
It might be good to allow people to change the heirarchy of roles, like this:

![role-hierarchy](https://user-images.githubusercontent.com/3868028/69651324-e5a3fb00-103d-11ea-88ce-1bda49256abf.png)

**Comment by stevejburge** [comment]
This could go into a Settings tab: https://github.com/publishpress/PublishPress-Capabilities/issues/31

Let's hold off on this until the main Capabilities screen is done
https://github.com/publishpress/PublishPress-Capabilities/issues/32#issuecomment-596607386

**Comment by htmgarcia** [comment]
@stevejburge how to reach this section?
![role-hierarchy](https://user-images.githubusercontent.com/3868028/69651324-e5a3fb00-103d-11ea-88ce-1bda49256abf.png)

**Comment by stevejburge** [comment]
@htmgarcia It doesn't exist yet :) I'll add it to a milestone

**Comment by olatechpro** [comment]
@stevejburge permit my ignorance. What's roles hierarchy, is it same as level and what influence does this have on roles/what are we changing in role? I have the feelings this idea is out of fashion in WordPress.

Also, where are we adding this control?
